src/iexamine.cpp

```diff
@@ -3497,6 +3498,80 @@ void iexamine::climb_down( player &p, const tripoint &examp )
     g->m.creature_on_trap( p );
 }
 
+void iexamine::autodoc( player &p, const tripoint &examp ) {
+    enum options {
+        CONFIGURE,
+        INSTALL_CBM,
+        CANCEL,
+    };
+
+    std::string status = uistate.iexamine_autodoc_config ? _( "Online" ) :
+                         _( "Configuration required" );
+
+    uimenu amenu;
+    amenu.selected = 0;
+    amenu.text = string_format( _( "Autodoc Mk. XI. Status: %s. Please choose operation." ), status );
+    amenu.addentry( CONFIGURE, true, 'c', str_to_illiterate_str( _( "Configure Autodoc." ) ) );
+    amenu.addentry( INSTALL_CBM, uistate.iexamine_autodoc_config ? true : false, 'i',
```

Comment:
```uistate.iexamine_autodoc_config ? true : false```
can be effectively shorten to just ```uistate.iexamine_autodoc_config``` since it's already a boolean variable.